qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -1025,16 +1018,17 @@ def new_xy_concatenated_sequence(
         <https://doi.org/10.1103/PhysRevLett.95.180501>`_
 
     """
-    duration = _check_duration(duration)
 
-    concatenation_order = concatenation_order or 1
+    check_arguments(
+        duration > 0, "Sequence duration must be above zero.", {"duration": duration}
+    )
+    check_arguments(
+        concatenation_order > 0,
+        "Concatenation oder must be above zero:",
+        {"concatenation_order": concatenation_order},
+    )
+
     concatenation_order = int(concatenation_order)
```

Comment:
If `concatentation_order` is between 0 and 1 (exclusive) then the `check_argument` will pass but `concatenation_order` will still end up as zero. I'd change the check to `>= 1`.

Reply:
fixed